Mage.Sets/src/mage/cards/e/ErietteOfTheCharmedApple.java

```diff
@@ -0,0 +1,94 @@
+package mage.cards.e;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.BeginningOfEndStepTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.LoseLifeOpponentsEffect;
+import mage.abilities.effects.common.combat.CantAttackYouAllEffect;
+import mage.abilities.hint.Hint;
+import mage.abilities.hint.ValueHint;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.permanent.ControllerIdPredicate;
+import mage.filter.predicate.permanent.EnchantedByPlayerPredicate;
+import mage.game.Game;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class ErietteOfTheCharmedApple extends CardImpl {
+
+    private FilterCreaturePermanent filter = new FilterCreaturePermanent("creatures enchanted by an Aura you control");
+    private final Hint hint = new ValueHint("Number of Auras you control", ErietteOfTheCharmedAppleValue.instance);
+
+    public ErietteOfTheCharmedApple(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}{B}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.WARLOCK);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(4);
+
+        // Each creature that's enchanted by an Aura you control can't attack you or planeswalkers you control.
+        filter.add(new EnchantedByPlayerPredicate(ownerId));
```

Comment:
This seems like something really unorthodox (and probably does not work on control change).
See the comment on the Predicate code for something that is more like other code pieces.